transports/quic/Cargo.toml

```
@@ -0,0 +1,42 @@
+[package]
+name = "libp2p-quic"
+version = "0.7.0"
```

```suggestion
version = "0.7.0-alpha"
```

I suggest we merge a first version as `alpha`. I think this better sets expectations while allowing folks to experiment with the new transport earlier.

What do you think?